flashmodels/patch/patch.py

```diff
@@ -34,36 +28,25 @@ def rewrite_load():
     exec(modified, transformers.modeling_utils.__dict__)
 
 
-def patch_llama():
-    transformers.models.llama.modeling_llama._make_causal_mask = make_causal_mask
-    if os.getenv("ACC_FLASH_ATTN", "0") == "1":
-        transformers.models.llama.modeling_llama.LlamaModel._prepare_decoder_attention_mask = flash_attn_prep_mask
-        transformers.models.llama.modeling_llama.LlamaAttention.forward = flash_attn_fwd
-    elif os.environ.get("ACC_LLAMA_TP") == "1":
-        transformers.models.llama.modeling_llama.LlamaMLP = LlamaMLP
-    if os.getenv("XLA_USE_SPMD") == "1":
-        # use einsum in linear for SPMD TP/Ulysses.
-        transformers.models.llama.modeling_llama.LlamaAttention = LlamaAttention
-        transformers.models.llama.modeling_llama.LlamaDecoderLayer = LlamaDecoderLayer
+def patch_llama(use_flash_attn):
```

Comment:
what if the transformers lib wants to use this function? Could we put this in torchacc